inc/settings.php

```
@@ -7,14 +7,92 @@
+ *
+ * @param mixed $value Raw providers value.
+ * @return array Sanitized providers list.
+ */
+function sanitize_providers_list( $value ): array {
+	if ( ! is_array( $value ) ) {
+		return [];
+	}
+
+	$sanitized = [];
+	foreach ( $value as $provider ) {
+		if ( is_array( $provider ) {
```

PHPStan (2.1.46)

[error] Syntax error, unexpected '{' on line 68


(phpstan.parse)